cola/widgets/standard.py

```diff
@@ -496,6 +496,12 @@ def __init__(self, parent=None):
             flags = self.windowFlags() & ~help_hint
             self.setWindowFlags(flags)
 
+    def setLayout(self, layout):
+        frame = QtWidgets.QFrame(self)
+        frame.setLayout(layout)
+        main_layout = qtutils.vbox(0, 0, frame)
+        self.Base.setLayout(self, main_layout)
+
```

Comment:
This is because of dialog windows has none margins at all, even in the current default theme. Look at the example:
![No margin dialog](https://i.imgur.com/oS9caNr.png)

And the reason I did it in setLayout method is that it was the only way to make it in one place, instead of changing every dialog. I will try to make it another way (eg changing margins of buttons etc.). What do you think?